frontend/assets/js/calculator.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', () => {
+    init();
+});
+
+function init() {
+    document.getElementById('split').addEventListener('click', () => {
+        split();
+    });
+
+    document.getElementById('clear').addEventListener('click', () => {
+        clear();
+    });
+}
+
+function split() {
+    let amount = document.getElementById('amount');
+    let people = document.getElementById('people');
+
+    let message = document.getElementById('message');
```

Comment:
I like your calculator. 
I am not 100% sure, but the document.getElementById('idName') returns the object. 
Therefor instead of let you can use const. You are not changing object itself but you are manipulating with its value /property. 

https://tylermcginnis.com/var-let-const/ -> here is one video at the end its explaining. 

The change would be:
    const amount = document.getElementById('amount');
    const people = document.getElementById('people');
    const message = document.getElementById('message');